collector/aws/resources/rds.go

```diff
@@ -136,9 +153,50 @@ func (r *RDSManager) Detect(metrics []config.MetricConfig) (interface{}, error)
 					"formula_value":       formulaValue,
 					"name":                *instance.DBInstanceIdentifier,
 					"instance_type":       *instance.DBInstanceClass,
+					"engine":              *instance.Engine,
 					"region":              r.awsManager.GetRegion(),
 				}).Info("RDS instance detected as unutilized resource")
 
+				instancePricingFilters := r.getPricingInstanceFilterInput(instance)
+				instancePrice, err := r.awsManager.GetPricingClient().GetPrice(instancePricingFilters, "", r.awsManager.GetRegion())
+				if err != nil {
+					log.WithError(err).Error("Could not get rds instance price")
+					continue
+				}
+
+				var hourlyStoragePrice float64
+				if rdsStorageType, found := rdsStorageType[*instance.StorageType]; found {
```

Comment:
what do you say to move this logic into a function? 
it will help us for better tests coverage + make `Detect` function to be more readable